sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3487,7 +3490,8 @@ def teardown_no_lock(self,
                          terminate: bool,
                          purge: bool = False,
                          post_teardown_cleanup: bool = True,
-                         refresh_cluster_status: bool = True) -> None:
+                         refresh_cluster_status: bool = True,
+                         allow_resource_not_found: bool = False) -> None:
```

Comment:
Why do we add this new argument? It seems we should always allow the termination of a non-existing cluster.

Reply:
When we `sky down` a cluster and its VM already been deleted on the cloud console, do we want to error out...? I'm not sure

Reply:
I don't think we should error out.

Reply:
🫡